contracts/factory/src/contract.rs

```diff
@@ -32,7 +29,10 @@ use drop_staking_base::{
         },
         withdrawal_voucher::InstantiateMsg as WithdrawalVoucherInstantiateMsg,
     },
-    state::pump::PumpTimeout,
+    state::{
+        factory::{State, STATE},
+        pump::PumpTimeout,
+    },
```

Comment:
this seem to be out of the scope of this PR

Reply:
What exactly?

Reply:
moving messages from factory to package

Reply:
also as far as I remember it was made in the other pr

Reply:
It's related because it's used in the contract itself to query a factory state

Reply:
Yeah, it was done in https://github.com/hadronlabs-org/drop-contracts/pull/81. I guess it's okay to do that in both of these PR's and merge updates between them in future

Reply:
oh, my bad. IC. so this way I see you are already trying to query factory in phonebook-like manner. this way I think it we be great to adopt [actual phonebook implementation](https://github.com/hadronlabs-org/drop-contracts/pull/204)